src/components/drop-down/drop-down.tsx

```diff
@@ -10,12 +11,34 @@ export interface DropDownInputProps {
 }
 
 export const DropDownInput: React.SFC<DropDownInputProps> = SBStateless(props => {
-  return (
-      <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
-          <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
-          <CaretDown className="caret" />
-      </div>
-  );
+    return (
+        <div data-automation-id="DROP_DOWN_INPUT" onClick={props.onClick} className="drop-down-input">
+            <span className="label">{props.selectedItem ? props.selectedItem.label : 'Default Text'}</span>
+            <CaretDown className="caret" />
+        </div>
+    );
+}, style);
+
+export interface DropDownListProps {
+    open: boolean;
+    items?: DropDownItem[];
```

Comment:
I understand. I will look into extending OptionList to have a more uniform interface across the components